Tests/CommandsTests/WorkspaceTests.swift

```diff
@@ -41,6 +41,9 @@ private class TestWorkspaceDelegate: WorkspaceDelegate {
     func checkingOut(repository: String, at reference: String) {
         print("Checking out: " + repository + " at " + reference)
     }
+
+    func removing(repository: String) {
+    }
```

Comment:
Ah, saw it. IMHO, its easier to review if test accompanies commit adding the feature...